mails/themes/modern/core/order_conf.html.twig

```diff
@@ -246,12 +246,14 @@
                                   <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="" width="100%">
                                     <tr>
                                       <td align="left" style="font-size:0px;padding:10px 25px;word-break:break-word;">
-                                        <div style="font-family:Open sans, arial, sans-serif;font-size:16px;line-height:25px;text-align:left;color:#363A41;"><span class="label">{{ 'Order:'|trans({}, 'Emails.Body', locale)|raw }}</span> {order_name} {{ 'Placed on'|trans({}, 'Emails.Body', locale)|raw }} {date}</div>
+                                        <div style="font-family:Open sans, arial, sans-serif;font-size:16px;line-height:25px;text-align:left;color:#363A41;">
```

Comment:
ALL modification to the modern theme must be performed via the generator repository https://github.com/PrestaShop/mjml-theme-converter/ where the MJML sources are located

Because this theme must be generated via our MJML tool to ensure it is correctly integrated, I documented this in the README but nobody seems to read it, and I can't blame them 😅. Maybe se should add an automatic warning via issue bot, because only I remember about this and I won't be able to catch all PR all the time

Besides this PR needs to be merged before you make your modif: https://github.com/PrestaShop/mjml-theme-converter/pull/7 since it also modifies this file

Reply:
Yep, that's exaclty what I did! https://github.com/PrestaShop/mjml-theme-converter/pull/9